Cargo.toml

```diff
@@ -5,4 +5,12 @@ members = [
 ]
 
 [profile.release]
+debug = false
+strip = "debuginfo"
+panic = "abort"
```

Comment:
This is quite harsh, it means that our app will completely die on a panic that could be caught by the webserver, and e.g. reported to sentry.
```suggestion
```